src/Misc/layoutbin/systemd.svc.sh.template

```diff
@@ -106,15 +106,27 @@ function stop()
 
 function uninstall()
 {
-    stop
-    systemctl disable ${SVC_NAME} || failed "failed to disable  ${SVC_NAME}"
-    rm "${UNIT_PATH}" || failed "failed to delete ${UNIT_PATH}"
+    if service_exists; then
```

Comment:
maybe just `if [ -f "${UNIT_PATH}" ]; then` which is similar to what we have in `function status()`

Reply:
I was also eyeing the `status` function, but wasn't sure if ${UNIT_PATH} is a good enough indicator - the whole problem is due to mismanaged `.service` files and it didn't feel right to rely on the one in $UNIT_PATH (`etc/systemd/system/`) to fix the one in $CONFIG_PATH.

I guess I'm just being paranoid, I'll move `if [ -f "${UNIT_PATH}" ]; then` to `service_exists` and have `status` rely on it (i.e. use `if service_exists; then` in `status`)

Reply:
sure, I just want to make sure we have a unified way to check service exists in this file